apps/mobile/src/components/MintLinkButton.tsx

```diff
@@ -65,7 +66,8 @@ export function MintLinkButton({
     token?.definition?.community?.contract?.contractAddress?.address ?? '';
   const tokenChain = token?.definition?.community?.contract?.contractAddress?.chain ?? '';
   const { url: mintURL, provider: mintProviderType } = getMintUrlWithReferrer(
-    overwriteURL || (token?.definition?.community?.contract?.mintURL ?? ''),
+    overwriteURL ||
+      (token?.definition?.mintUrl ?? token.definition?.community?.contract?.mintURL ?? ''),
```

Comment:
Added the fallback to `community` mintURL too, just in case the token dont have `mintUrl`

Reply:
great idea